.github/workflows/testBuildHybrid.yml

```diff
@@ -79,9 +74,9 @@ jobs:
 
             return found.trim();
 
-  getOldDotBranchRef:
+  getOldDotRef:
       runs-on: ubuntu-latest
-      needs: getOldDotPR
+      needs: [getOldDotPR]
```

Comment:
What does the square brackets do?

Reply:
They are used when you want to pass more than one dependency. It doesn't change anything in this case.